File: C/1006.c
#include <stdio.h>

int main() {
    float a, b, c;
    scanf("%f %f %f", &a, &b, &c);
    
    printf("MEDIA = %.1f\n", (a * 2 + b * 3 + c * 5) / 10);
    return 0;
}

File: Golang/1006.go
package main

import (
	"fmt"
)

func main() {
	var a, b, c float64

	fmt.Scanf("%f\n%f\n%f", &a, &b, &c)

	fmt.Printf("MEDIA = %.1f\n", (a*2+b*3+c*5)/10)
}
